### ISSUE TITLE
theme issue with storybook version 5

### ISSUE THREAD

**Issue by wonskarol** [vertexbz/storybook-addon-react-live-edit#8]
Does this plugin support version 5 of Storybook? 
I'm getting following error:
```
WARN FATAL broken build!, will close the process,
WARN Fix the error below and restart storybook.

ModuleNotFoundError: Module not found: Error: Can't resolve '@storybook/components/dist/theme' in '/Users/karolwons/ui-docs-next/node_modules/storybook-addon-react-live-edit/dist/component/LiveEditor'
```
Indeed there is no `theme` under `'@storybook/components/dist/theme' `

**Comment by Richacinas** [comment]
FYI, I'm using a fork of this repo that is already running with Storybook v5 successfully.

https://github.com/toshiaki61/storybook-addon-react-live-edit

**Comment by vertexbz** [comment]
Should be resolved in 2.0.4. Sorry for unavailability.
